test/src/e2e_vm_tests/test_programs/should_pass/test_contracts/basic_storage/src/main.sw

```diff
@@ -110,9 +122,7 @@ fn test_storage() {
     };
     store(key, s);
     let s_ = get::<S>(key);
-    assert(s.x == s_.x && s.y == s_.y && s.z == s_.z && s.t.x == s_.t.x
-           && s.t.y == s_.t.y && s.t.z == s_.t.z && s.t.boolean == s_.t.boolean
-           && s.t.int8 == s_.t.int8 && s.t.int16 == s_.t.int16 && s.t.int32 == s_.t.int32);
+    assert(s.x == s_.x && s.y == s_.y && s.z == s_.z && s.t.x == s_.t.x && s.t.y == s_.t.y && s.t.z == s_.t.z && s.t.boolean == s_.t.boolean && s.t.int8 == s_.t.int8 && s.t.int16 == s_.t.int16 && s.t.int32 == s_.t.int32);
```

Comment:
maybe we could put this on multiple lines 😅

Reply:
The formatter did this :( Yeah probably we should just break the `assert` into multiple ones.

Reply:
Addressed